rfcs/remote-client.md

```diff
@@ -0,0 +1,132 @@
+- Feature Name: Remote Client
+- Start Date: 2018-12-24 
+- RFC PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This feature allows for people using the `salt` CLI tool to connect remotely to
+any Salt master and to issue commands as though they were on the master itself.
+
+It removes the limitation for the CLI tool to have access to a stored key on the
+master and replaces that method with a public-key authentication system.
+
+# Motivation
+[motivation]: #motivation
+
+At present, if a person wishes to publish commands to a minion, they most often
+achieve this by logging into a master directly and then using the provided
+`salt` CLI tool. Altneratively, they may issue commands to a master by using
+`salt-api` which provides a series of REST endpoints which may be accessed
+natively, or by using an alternative CLI designed specifically for use
+with `salt-api`, which is known as `pepper`.
+
+However, even between these option, there remain several serious drawbacks:
+
+ * A user cannot use either the `salt` tool or the `pepper` tool to send
+   commands to a mix of API- and non-API- endpoints.
+ 
+ * Using `pepper`, there is no way to target a given master from the
```

Comment:
This is not true, pepper can have multiple salt-api endpoints configured in the pepperrc config file as different sections of the ini config, and that can be used to target different masters based on the profile selected to be used.

Reply:
Ah, I didn't know that. I will remove this section.